operator/config/samples/loggingplumber_v1alpha1_flowtest.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: flowtest-sample
```

Comment:
can we move this out of operator and into a top level folder like `develop`

Reply:
https://github.com/MrSupiri/rancher-logging-explorer/pull/31#discussion_r677701275